src/requests-from-cache.js

```diff
@@ -1,9 +1,6 @@
 // @flow
```

Comment:
@kevinbarabash the inclusion of these flowtypes should help make all this much clearer on how these calls get combined with others.